packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -113,7 +113,7 @@ export class AppStudioPluginImpl {
         return err(appDefinitionAndManifest.error);
       }
 
-      const localTeamsAppID = await this.getTeamsAppId(ctx, true);
+      const localTeamsAppID = await this.getTeamsAppId(ctx, !isConfigUnifyEnabled());
```

Comment:
The second parameter of `getTeamsAppId` is a Boolean flag to check if `isLocalDebug`, to avoid confusion and better readability, can we also update the signature for `getTeamsAppId`? We can remove the second parameter and check the flag inside the method implementation.

Reply:
I have made a mistake here, I cannot replace ```isLocalDebug``` parameter because it needs to read local manifest to get user input teams app ID. I add a ``TODO`` tag and will update this method without ```isLocalDebug```.